packages/components/src/date-time/date.js

```diff
@@ -1,135 +1,104 @@
 /**
  * External dependencies
  */
-import moment from 'moment';
-// react-dates doesn't tree-shake correctly, so we import from the individual
-// component here, to avoid including too much of the library
-import DayPickerSingleDateController from 'react-dates/lib/components/DayPickerSingleDateController';
+import ReactDatePicker from 'react-datepicker';
+import { isSameDay, format, setMonth, getMonth, getYear, getDate } from 'date-fns';
+import { map, filter } from 'lodash';
```

Comment:
I'd recommend using the native JS `[].map` and `[].filter` over the Lodash versions.

Reply:
👍